### PR TITLE
Enable display of cards search empty state

### PR DESCRIPTION
**Pull request by piedpiper358**
**Before:**

**After:**

### CODE REVIEW

File: core/src/js/components/collection/cards.component.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const DEFAULT_CARD_HEIGHT = 240;
 			<ng-container *ngIf="{ highRes: highRes$ | async } as value">
 				<virtual-scroller
 					#scroll
-					*ngIf="cards$ | async as activeCards; else emptyState"
+					*ngIf="(cards$ | async)?.length && (cards$ | async) as activeCards; else emptyState"
```
Comment (sebastientromp) [inline core/src/js/components/collection/cards.component.ts:31]: Here I would recommend to use the same trick as in #162

**Review by sebastientromp — CHANGES_REQUESTED** [review]

**Comment by piedpiper358** [comment]
Something like that?

**Comment by sebastientromp** [comment]
Yes :)
I'll merge that once tonight's patch aftermath is over

**Comment by sebastientromp** [comment]
This branch now has conflicts, would you like to look at them? Otherwise I'll have a look myself

**Comment by piedpiper358** [comment]
Done

File: core/src/js/components/collection/cards.component.ts [hunk 2]
```diff
@@ -18,18 +18,18 @@ export const DEFAULT_CARD_HEIGHT = 240;
 	selector: 'cards',
 	styleUrls: [`../../../css/component/collection/cards.component.scss`, `../../../css/global/scrollbar.scss`],
 	template: `
-		<div class="cards">
-			<div class="show-filter">
+		<div class="cards" *ngIf="{ cards: cards$ | async } as value">
+			<div class="show-filter" [ngStyle]="{ 'display': _searchString ? 'none' : 'flex' }">
```
Comment (sebastientromp) [inline core/src/js/components/collection/cards.component.ts:22]: I think we need to show the filters, as they influence the search results

Comment (piedpiper358) [reply core/src/js/components/collection/cards.component.ts:22]: Yes, of course, this is me rushing.

**Pull request merged**